File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/RetailerProfile.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class RetailerProfile 
{
    public string DetailsHeader { get; set; }
    public string DetailsBody { get; set; }
    public string DetailsLineLeft1 { get; set; }
    public string DetailsLineLeft2 { get; set; }
    public string DetailsLineRight1 { get; set; }
    public string MetaDescription { get; set; }
    public string Directions { get; set; }
    public string Hours { get; set; }
    public string HomePhone { get; set; }
    public string CellPhone { get; set; }
    public CellCarrier CellCarrier { get; set; }
    public string BusinessPhone { get; set; }
    public string BusinessPhoneExt { get; set; }
    public string FaxNumber { get; set; }
    public string VarTimeZone { get; set; }
    public string UtcOffset { get; set; }
    public string Code501c3 { get; set; }
    public string EmailAddress { get; set; }
    public Address Address { get; set; }
    public string Web { get; set; }
    public bool Featured { get; set; }
    public bool Listed { get; set; }
}

File: server-scala-play-server/app/model/RetailerProfile.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for RetailerProfile.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class RetailerProfile(
  detailsHeader: Option[String],
  detailsBody: Option[String],
  detailsLineLeft1: Option[String],
  detailsLineLeft2: Option[String],
  detailsLineRight1: Option[String],
  metaDescription: Option[String],
  directions: Option[String],
  hours: Option[String],
  homePhone: Option[String],
  cellPhone: Option[String],
  cellCarrier: Option[CellCarrier],
  businessPhone: Option[String],
  businessPhoneExt: Option[String],
  faxNumber: Option[String],
  timeZone: Option[String],
  utcOffset: Option[String],
  code501c3: Option[String],
  emailAddress: Option[String],
  address: Option[Address],
  web: Option[String],
  featured: Option[Boolean],
  listed: Option[Boolean]
)

object RetailerProfile {
  implicit lazy val retailerProfileJsonFormat: Format[RetailerProfile] = Json.format[RetailerProfile]
}

